packages/about-you/api-client/tests/api/removeFromCart/index.spec.ts

```diff
@@ -0,0 +1,18 @@
+import removeFromCart from '../../../src/api/removeFromCart';
+import { apiClient } from '../../../src/index';
+
+const mockDeleteItem = jest.fn((params) => params);
+
+apiClient.basket.deleteItem = mockDeleteItem;
```

Comment:
This isn't really how you should mock apiClient. I suggest to use jest.mock instead.